index.html

```diff
@@ -4,6 +4,8 @@
 <head>
   <meta charset="utf-8" />
   <link rel="icon" href="/favicon.ico" />
+  <script src="https://kit.fontawesome.com/b41738187f.js" crossorigin="anonymous"></script>
```

Comment:
I think we can remove this line. We have `react-icons` installed and are already using them, so maybe we can use that instead.

Reply:
Yes, I will do it.